src/breadcrumb-group/internal.tsx

```diff
@@ -33,6 +34,7 @@ const DropdownTrigger = (
       ref={ref}
       aria-expanded={isExpanded ? true : undefined}
       aria-haspopup={true}
+      ariaLabel={ariaLabel}
```

Comment:
could you also change button type from 'submit' to 'button' like mentioned in the ticket?

Reply:
Done